outputs.tf

```diff
@@ -3,6 +3,11 @@ output "lb_dns" {
   description = "The DNS value of your LB hosting the concourse cluster. Point your FQDN to it."
 }
 
+output "lb_zone_id" {
+  value       = aws_elb.concourse_lb.zone_id
+  description = "The zone ID of your LB hosting the concourse cluster."
+}
```

Comment:
Ah, this is unrelated to the purpose of this PR. However, it is something that can be useful for module clients.